src/Command/DispatchEventsCommand.php

```diff
@@ -0,0 +1,132 @@
+   * @param TransportNameServiceInterface $transportNameService
+   * @param array                         $transportEventInformationMapping
+   * @param bool                          $enabled
+   */
+  public function __construct(EventDispatcherInterface $eventDispatcher,
+                              QueueEventServiceInterface $queueEventService,
+                              TransportNameServiceInterface $transportNameService,
+                              array $transportEventInformationMapping,
+                              bool $enabled) {
+    parent::__construct();
+    $this->eventDispatcher = $eventDispatcher;
+    $this->queueEventService = $queueEventService;
+    $this->transportNameService = $transportNameService;
+    $this->transportEventInformationMapping = $transportEventInformationMapping;
+    $this->enabled = $enabled;
+  }
+
+  /**
+   * @inheritDoc
+   */
+  protected function configure(): void {
+    $this->setDescription('Dispatch message queue events')
+      ->addArgument('transports', InputArgument::IS_ARRAY, 'transports you want to dispatch events for', []);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  protected function execute(InputInterface $input, OutputInterface $output): int {
+    $io = new SymfonyStyle($input, $output);
+
+    if (!$this->enabled) {
+      $io->warning('queue events is not enabled');
+      return Command::SUCCESS;
+    }
+
+    $dispatchedCount = 0;
+    $transportNames = $this->transportNameService->getTransportNames();
+    foreach ($transportNames as $transportName) {
+      $eventInformationTypes = $this->transportEventInformationMapping[$transportName] ?? MessageQueueEvent::EVENT_INFORMATION_TYPES;
+
+      if (empty($eventInformationTypes)) {
+        if ($io->isVerbose()) {
+          $io->note(\sprintf('Transport %s have no information types mapped, not sending a event.', $transportName));
+        }
+
+        continue;
+      }
+
+      if ($io->isVerbose()) {
+        $io->note(\sprintf('Dispatching event for transport %s', $transportName));
+      }
+
+      $this->eventDispatcher->dispatch($this->createEvent($transportName, $eventInformationTypes));
+      $dispatchedCount++;
+    }
+
+    $io->success(\sprintf('Dispatched %d events', $dispatchedCount));
+    return Command::SUCCESS;
+  }
+
+  /**
+   * @param string $transport
+   * @param array  $eventInformationTypes
+   *
+   * @return MessageQueueEvent
+   */
+  private function createEvent(string $transport, array $eventInformationTypes): MessageQueueEvent {
+    $event = new MessageQueueEvent($transport, $eventInformationTypes);
+    $this->setInformation($event, $eventInformationTypes);
+
+    return $event;
+  }
+
+  /**
+   * @param MessageQueueEvent $event
+   * @param array             $eventInformationTypes
+   */
+  private function setInformation(MessageQueueEvent $event, array $eventInformationTypes): void {
+    foreach ($eventInformationTypes as $eventInformationType) {
```

Comment:
I'm not sure if it is possible but if all of the elements of the array `$eventInformationTypes` are different types anyway, wouldn't it be better to have an object containing them instead of an array?